src/sage/graphs/graph_database.py

```diff
@@ -180,18 +180,18 @@ def subgraphs_to_query(subgraphs, db):
 
 # tables     columns                    input data type     sqlite data type
 # -----------------------------------------------------------------------------
-aut_grp =  ['aut_grp_size',             # Integer           INTEGER
+aut_grp = ['aut_grp_size',             # Integer           INTEGER
             'num_orbits',               # Integer           INTEGER
             'num_fixed_points',         # Integer           INTEGER
             'vertex_transitive',        # bool              BOOLEAN
             'edge_transitive']          # bool              BOOLEAN
```

Comment:
Well.. Then the following lines should be adjusted to be aligned right.

Reply:
indeed, done